docs/client/install-kn.md

```diff
@@ -8,6 +8,15 @@ aliases:
 
 This guide provides details about how you can set up the Knative `kn` CLI.
 
+## Install kn using brew
```

Comment:
Not sure why we'd move this, binary seems more universal to me?

Reply:
It's just the lowest friction way to install kn, if someone doesn't have homebrew they will keep scrolling and find the binary. 

But if they do have homebrew installed they can effortlessly install the CLI. Seems like a standard across many Dev Tools I have personally used?

Reply:
Yes but it requires an additional tool. Personally I don't use it and as a user I would want the options first that don't require additional tools; can we do some user research about stuff like this? Otherwise it's just us agreeing over personal opinions 😂

Reply:
Happily! I'll add it to the user testing next week

Reply:
Can we push this and I pinky promise to report back :)

Reply:
Yeah I can live with it 😄